env.example

```diff
@@ -315,6 +315,12 @@ JIBRI_LOGS_DIR=/config/logs
 # Disable HTTPS: handle TLS connections outside of this setup
 #DISABLE_HTTPS=1
 
+# See https://github.com/jitsi/jitsi-meet/blob/master/config.js for defaults.
+#CONFIG_JSON={disableThirdPartyRequests: true}
```

Comment:
This is not JSON but actual JS.

Reply:
Do you have a suggestion?

Reply:
You could get the value quoted, as validd JSON and use JSON.parse on it when you assign it.

Reply:
Ah.. I though about just rename the variable like something without JSON. I don't see the benefit to use `JSON.parse`.